src/downloadmanagement.h

```diff
@@ -207,3 +207,5 @@ class DownloadManager : public QObject
 };
 
 #endif // DOWNLOADMANAGEMENT_H
+
+std::string getDataDirectory();
```

Comment:
I don't think that this function should be declared in this header file, but if there is no better place then it should at least be moved to before the `#endif`

Reply:
I think that `settingsmanager.h/cpp` is a more appropriate home for this function's declaration and definition.